lp-builder-config/sync.log

```diff
@@ -0,0 +1,357 @@
+Checking charm recipes for charm Ceph Dashboard Charm
+Charm recipe charm-ceph-dashboard.master.latest has changes. Saving.
+Changes: recipe.auto_build_channels = {'charmcraft': '1.5/stable'}
+Charm recipe charm-ceph-dashboard.master.quincy has changes. Saving.
```

Comment:
I think this file has been added in error?

Reply:
indeed, fixing it now.